src/sagemaker/image_uri_config/xgboost.json

```diff
@@ -281,6 +281,40 @@
                     "us-west-2": "246618743249"
                 },
                 "repository": "sagemaker-xgboost"
+            },
+            "1.7-1": {
+                "registries": {
+                    "af-south-1": "510948584623",
+                    "ap-east-1": "651117190479",
+                    "ap-northeast-1": "354813040037",
+                    "ap-northeast-2": "366743142698",
+                    "ap-northeast-3": "867004704886",
+                    "ap-south-1": "720646828776",
+                    "ap-southeast-1": "121021644041",
+                    "ap-southeast-2": "783357654285",
+                    "ap-southeast-3": "951798379941",
```

Comment:
I think our recent deployments actually added 1.7-1 to the 4 new regions as well. Could you double check (i.e. look into the ECR accounts of the new regions for this tagged image) and then add them here as well? Thanks!

Reply:
Amending my comment slightly, since I noticed we're not in the staging repo. Could you either make a PR to the staging repo for the change I mentioned above, or a backlog item for us to do so ? Thanks!

Reply:
Good point! I'll create a backlog for launching all versions of XGB and SKLearn in those 4 regions.